terraform/etl/48-lambda-gov-notify-ingestion.tf

```
@@ -120,3 +128,48 @@ resource "aws_iam_role_policy_attachment" "gov_notify_lambda_secret_access" {
   role       = aws_iam_role.housing_gov_notify_ingestion[0].name
   policy_arn = aws_iam_policy.gov_notify_lambda_secret_access[0].arn
 }
+
+
+module "gov-notify-ingestion-housing-repairs" {
+  count                          = !local.is_production_environment ? 1 : 0
```

and to get everything in sync this should be 

```suggestion
  count                          = local.create_govnotify_resource_count
```